src/app/src/mainwindow.cpp

```diff
@@ -160,6 +161,7 @@ MainWindow::MainWindow(std::shared_ptr<Config> params, QMainWindow *parent) : QM
 
     // plate ID widget
     m_db = new Database(m_config->userProfile);
+    m_db->initDB(); // TODO what to do here if this fails?
```

Comment:
I think it should raise the pop-up like it does when the camera can't connect, and exit once the user acknowledges the error